lua/xcodebuild/helpers.lua

```diff
@@ -72,8 +72,18 @@ end
 ---@return table<string, string[]>
 function M.find_all_swift_files()
   local util = require("xcodebuild.util")
-  local allFiles =
-    util.shell("find '" .. vim.fn.getcwd() .. "' -type f -iname '*.swift' -not -path '*/.*' 2>/dev/null")
+
+  local allFiles
+  if util.is_fd_installed() then
+    allFiles = util.shell("fd -I '.*\\.swift$' '" .. vim.fn.getcwd() .. "' --type f 2> /dev/null")
+  else
+    allFiles = util.shell(
+      "find '"
+        .. vim.fn.getcwd()
+        .. "' -type d -path '*/.*' -prune -false -o -type f -iname '*.swift' 2>/dev/null"
```

Comment:
Following my last comment it should be:
`' -type d -path '*/.*' -prune -o -type f -iname '*.swift' -print 2>/dev/null`